src/Components/Reports/Common/ExecutiveThirdPage.js

```diff
@@ -0,0 +1,99 @@
+import React, { Fragment } from 'react';
+import { Text } from '@react-pdf/renderer';
+import { TachometerAltIconConfig } from '@patternfly/react-icons/dist/js/icons/tachometer-alt-icon';
+import { AngleDoubleDownIconConfig } from '@patternfly/react-icons/dist/js/icons/angle-double-down-icon';
+import { AngleDoubleUpIconConfig } from '@patternfly/react-icons/dist/js/icons/angle-double-up-icon';
+import { AutomationIconConfig } from '@patternfly/react-icons/dist/js/icons/automation-icon';
+import { InProgressIconConfig } from '@patternfly/react-icons/dist/js/icons/in-progress-icon';
+import { CheckCircleIconConfig } from '@patternfly/react-icons/dist/js/icons/check-circle-icon';
+import styles from './styles';
+import { DescriptionList } from './DescriptionList';
+
+export const ExecutiveThirdPage = () => {
+
+    const stateDescription = [
```

Comment:
@kgaikwad thanks for suggesting the variable name, I will refactor it. Regarding moving out it a separate file, I can do it but  we will still have to refactor the page name in case we add any other page in future. Let me know your thoughts or if I missed something in understanding the same 🙌🏻